src/components/Icons.tsx

```diff
@@ -559,7 +561,9 @@ export const ItemIconMap = {
     CRASH1000: Wrapper(Crash1000),
     RDBEAR: Wrapper(MarketBear),
     RDBULL: Wrapper(MarketBull),
-    stpRNG: Wrapper(Step),
+    stpRNG: Wrapper(Step100),
+    stpRNG2: Wrapper(Step200),
```

Comment:
Please ignore if irrelevant . If there is a reason the name `stpRNG` is in Uppercase in deriv-app and camelCase here?

Reply:
@ahmadtaimoor-deriv the reason is that here it's taken from `active_symbols` response while in deriv-app - from `shortcode` where it's in camel case)

Reply:
Thanks